src/coord/src/timestamp.rs

```diff
@@ -199,57 +199,56 @@ struct ByoFileConnector<Out, Err> {
     stream: Receiver<Result<Out, Err>>,
 }
 
-fn byo_query_source(consumer: &mut ByoTimestampConsumer) -> Vec<ValueEncoding> {
+fn byo_query_source(consumer: &mut ByoTimestampConsumer) -> anyhow::Result<Vec<ValueEncoding>> {
```

Comment:
the convention in the rest of the code is to use the std library `Result` type and specify `anyhow::Error` manually

Reply:
Done!